react/features/filmstrip/components/web/ThumbnailWrapper.js

```diff
@@ -151,18 +151,22 @@ function _mapStateToProps(state, ownProps) {
         // When the thumbnails are reordered, local participant is inserted at index 0.
         const localIndex = enableThumbnailReordering && !disableSelfView ? 0 : remoteParticipantsLength;
         const remoteIndex = enableThumbnailReordering && !iAmRecorder && !disableSelfView ? index - 1 : index;
+        const _isAnyParticipantPinned = _verticalViewGrid
+            && Boolean([ ...remote ].find(([ , value ]) => value?.pinned) || local?.pinned);
```

Comment:
Won't this be expensive to do with 500 participants?

Reply:
Hmm, we already do it for the vertical filmstrip, but it's a good point, this could be optimized

Reply:
Done